buddyboss/buddyboss-platform
# Fix discussion reply with document not loading when documents upload disabled for activity

Discussion reply with a document not loading when document upload disabled for activity and profile.

### All Submissions:






### Changes proposed in this Pull Request:

It was not working because the default type is `profile` in the `bb_user_has_access_upload_document( $group_id = 0, $user_id = 0, $forum_id = 0, $thread_id = 0, $type = 'profile' )` function and we are not passing the forum while editing the forum. So it was taking the `profile` as the default argument. That is the reason for the issue. 

Fixes #2798 .

### How to test the changes in this Pull Request:

1. Uncheck this option from buddyboss plugin setting.
![screenshot-buddyboss test-2021 08 19-15_58_35](https://user-images.githubusercontent.com/25550562/130054263-9ccdfa04-06d4-43aa-99c1-412a5210652a.png)
2. Not add a document in the forum.
3. Try to edit that reply with a document.
4. See Document is visible.

### Proof Screenshots or Video
![screenshot-buddyboss test-2021 08 19-16_05_00](https://user-images.githubusercontent.com/25550562/130054523-cdd5594b-9977-4da2-b7c6-1a48343d6b18.png)


### Other information:
